IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -192,6 +207,10 @@ - (MSIDWebviewConfiguration *)webViewConfigurationWithRequestParameters:(MSIDInt
 #if TARGET_OS_IPHONE
     configuration.parentController = parameters.parentViewController;
     configuration.presentationType = parameters.presentationType;
+    if (@available(iOS 13.0, *))
```

Comment:
wouldn't it also be applicable for macOS?

Reply:
macOS will be addressed in separate PR.

Reply:
shouldn't we target iOS 13 **and above**? :)

Reply:
Oh, @available is doing exactly that. Ignore me :P